lib/cli/input/eval.js

```diff
@@ -0,0 +1,71 @@
+const inquirer = require('inquirer');
+const PromptGenerator = require('../../utils/PromptGenerator');
+const preferenceManager = require('../../utils/preference-manager');
+const path = require('path');
+
+const defaultPrefPath = path.join(__dirname, '../../../default-prefs.json');
+const defaultPrefs = JSON.parse(require('fs').readFileSync(defaultPrefPath, 'utf8'));
+const {supportedLanguages} = defaultPrefs;
+
+const getLengthValidator = (invalidMessage) => {
+  return (value) => {
+    if (value.length) {
+      return true;
+    }
+    else {
+      return invalidMessage;
+    }
+  }
+}
+
+const getLabPrompt = async () => {
```

Comment:
This arrow function is too big. It may be better to make this a proper function.